src/components/EnterpriseApp/index.jsx

```diff
@@ -212,6 +214,7 @@ EnterpriseApp.propTypes = {
   enableAnalyticsScreen: PropTypes.bool,
   enableReportingConfigurationsScreen: PropTypes.bool,
   enablePortalLearnerCreditManagementScreen: PropTypes.bool,
+  enterpriseGroupsV2: PropTypes.bool,
```

Comment:
I don't think we need this prop here. it's an field under `enterpriseFeatures` object which I see has already been added to line 199.